### ISSUE TITLE
Bug: convertLatexToMathMl not transforming properly extensible accents

### ISSUE THREAD

**Issue by FrancoRossi-Globant** [arnog/mathlive#2134]
## Description

> I need to convert Latex formulas to MathML with the ```convertLatexToMathMl(string) ```. Currently there are a few Extensible Accents formulas that are not being correctly converted.

In this [codepen](https://codesandbox.io/s/convertlatextomathml-yhhjr2)i've added a couple of Extensible Accents convertion adding its return value to the doc. In mathfield im showing how we need to be render with the exact formula.

### Steps to Reproduce

> Provide steps that are specific and repeatable

1. add function with a call to convertLatexToMathMl.
2. on convertLatexToMathMl add a string param of ``` '\\overgroup{ab}') ``` ej: ````convertLatexToMathMl('\\overgroup{ab}');```
3. log/print result of called function.

### Actual Behavior

> The return of convertLatexToMathMl function isn't returning expected math tags of Extensible Accents.

### Expected Behavior

> convertLatexToMathMl function to return expected math tags of Extensible Accents.

### Environment

**Operating System** Windows

**Browser** _Chrome, Firefox ... didnt test in others.

**Comment by arnog** [comment]
Thanks for providing the CodePen, that's useful.

However, I'm not sure what the "expected math tags" are. Could you elaborate on what you are expecting?

**Comment by FrancoRossi-Globant** [comment]
Yes sorry, that wasn't very tech. I was referring to the conversion from latex to mathml, "math tags" would be mathml.
Taking for example "\overbrace{ab}" the conversion to mathml would be sth like...
```
<math display="block">
  <mover accent="true">
    <mrow>
      <mi>A</mi>
      <mi>B</mi>
    </mrow>
    <mo>&#x23DE</mo>
  </mover>
</math>
```

instead im getting:
```
<mover >⏞</mover>
```

**Comment by arnog** [comment]
OK, got it. You don't know where I could find a mapping of the LaTeX commands to the matching MathML Unicode operators by any chance?

**Comment by NSoiffer** [comment]
Take a look at https://github.com/joom/latex-unicoder.vim/blob/master/autoload/unicoder.vim

**Comment by arnog** [comment]
Thanks, @NSoiffer. That's a start, it seems to give the following:

- `\overbrace`: U+23DE
- `\overparen`: U+23DC

But there are several missing, include `\overgroup`, `\overline`, `\overlinesegment`, `\overrightarrow`, `\overleftarrow`, `\overarc`, `\wideparen` and `\widetilde.

And some are only provided with combining characters: `\overline`, `\overleftrightarrow` which seems like it's an error.

**Comment by NSoiffer** [comment]
I can't vouch for the accuracy of the list. Sorry.

For MathML, the over arrows would simply be `mover` with the appropriate arrow as the second arg. Same for a line (U+005F).

I don't know if there is a stretchy/wider tilde or arc.

**Comment by arnog** [comment]
The MathML operator dictionary does list some accents: see https://www.w3.org/TR/2009/WD-MathML3-20090924/appendixc.html, operators of precedence 880, bottom of the table.

**Comment by NSoiffer** [comment]
In case it isn't clear, you can click on a column and it will sort by that column. So clicking on "properties" is a way to get all the stretch operators together. Of course, whether a character actually will stretch depends upon the font being used.

**Comment by FrancoRossi-Globant** [comment]
Any update on this? Or any other help I could bring?

**Comment by arnog** [comment]
It could help if you could provide the specific mappings you are looking for.

**Comment by FrancoRossi-Globant** [comment]
Hi Arno, a long time without coming here.
What are you referring to the mapping??? 
An example here is that modifing somelines of code in the lib I get something more useful from the \overleftrightarrow.
Modified lines 
atom-to-math-ml.ts: 
``` 864: overscript = atom.above || atom.svgAbove; 
    866: if ((overscript) && (atom.svgBelow || underscript))
 ```
And I start getting this: '<mover ><mrow><mi>A</mi><mo>&#8290;</mo><mi>b</mi></mrow>←</mover>'
And what I want to get is this: ' <mover><mrow><mi>A</mi><mi>b</mi></mrow><mo stretchy="true" style="math-style: normal; math-depth: 0;">←</mo></mover>' see: https://temml.org/
